Apps/phipqualityscore/ipqualityscore.json

```diff
@@ -0,0 +1,1139 @@
+    "publisher": "IPQualityScore",
+    "package_name": "phantom_ipqualityscore",
+    "type": "investigative",
+    "license": "Copyright (c) 2021 IPQualityScore.com.",
+    "main_module": "ipqualityscore_connector.py",
+    "app_version": "1.0.0",
+    "utctime_updated": "2021-02-26T01:17:12Z",
+    "product_vendor": "IPQualityScore",
+    "product_name": "IPQualityScore",
+    "product_version_regex": ".*",
+    "min_phantom_version": "4.9.39220",
+    "python_version": "3",
+    "latest_tested_versions": [
+        "Cloud, API: ipqualityscore.com, February 25 2021"
+    ],
+    "logo": "logo_ipqualityscore.svg",
+    "logo_dark": "logo_ipqualityscore_dark.svg",
+    "configuration": {
+        "apikey": {
+            "data_type": "password",
+            "description": "API key",
+            "order": 0,
+            "required": true
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "description": "Validates the connectivity by querying IPQualityScore",
+            "type": "test",
+            "identifier": "test_asset_connectivity",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "email validation",
+            "description": "Queries IPQualityScore's Email Validation API",
+            "verbose": "If email information is unavailable in IPQualityScore, only 'email' and 'message' property would be populated.",
+            "type": "investigate",
+            "identifier": "email_validation",
+            "read_only": true,
+            "parameters": {
+                "email": {
+                    "required": true,
+                    "description": "Email to query for reputation information",
+                    "data_type": "string",
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "order": 0
+                },
+                "fast": {
+                    "description": "Enables or disables SMTP check with the mail service provider",
+                    "data_type": "boolean",
+                    "order": 1
+                },
+                "suggest_domain": {
+                    "description": "Force analyze if the email address's domain has a typo and should be corrected to a popular mail service",
+                    "data_type": "boolean",
+                    "order": 2
+                },
+                "timeout": {
+                    "description": "Maximum number of seconds to wait for a reply from a mail service provider",
+                    "data_type": "numeric",
+                    "order": 3
+                },
+                "strictness": {
+                    "description": "Sets how strictly spam traps and honeypots are detected by system",
```

Comment:
Possibly add that a higher or lower number indicates how strict

Reply:
@yanky076 also, could you please help us here?

Reply:
What are the accepted values and what do they mean?

Reply:
@yanky076, could you please help us here?